static/apps/style/views/right/marker-style-view.js

```diff
@@ -25,11 +25,7 @@ define(["jquery",
             },
             catDataHasBeenBuilt: false,
             template: Handlebars.compile(MarkerStyleTemplate),
-            modelEvents: {
-                //'change:symbols': 'render'//,
-                //'change:metadata': 'contData'
-            },
-
+            
```

Comment:
It might make sense, down the line, to break this up into smaller helper views.  ~600 lines for a few is a bit long. That said, it's working well!